signer/tuf_on_ci_sign/_signer_repository.py

```diff
@@ -760,9 +793,13 @@ def sign(self, rolename: str):
         md = self.open(rolename)
         signing_keys: dict[str, Key] = {}
         for key in self._get_keys(rolename):
-            keyowner = key.unrecognized_fields["x-tuf-on-ci-keyowner"]
-            if keyowner == self.user.name:
+            # Detect if the key is an online key
+            if TAG_ONLINE_URI in key.unrecognized_fields:
                 signing_keys[key.keyid] = key
+            else:
+                keyowner = key.unrecognized_fields[TAG_KEYOWNER]
+                if keyowner == self.user.name:
+                    signing_keys[key.keyid] = key
```

Comment:
Is this necessary here? We're trying to sign online keys in repo tools not in signer

Reply:
It appears it's a left over from an earlier version, good catch. I tried locally with this hunk reverted and it works. Removing this.